Gemfile

```diff
@@ -185,3 +185,6 @@ if ENV["IMPORT"] == "1"
   gem 'ruby-bbcode-to-md', github: 'nlalonde/ruby-bbcode-to-md'
   gem 'reverse_markdown'
 end
+
+gem 'hkdf', '0.3.0'
```

Comment:
I think you can require false these libs and only load them on demand in sidekiq